docs/examples/SPA.md

```diff
@@ -0,0 +1,66 @@
+## Using offline-plugin in a Single Page Application (SPA)
+
+In a SPA, only the inital HTML is served from the server. Any routing beyond that takes place on the client. So how should we configure offline-plugin to handle this?
```

Comment:
Also, I am a bit wary of using `index.html` as an example, since the server might deliver generated html. In case of a react app for example, it's a normal case.